components/src/Tables/Table.js

```diff
@@ -25,6 +26,54 @@ export const Cell = styled.td`
 
 Cell.defaultProps = defaultProps;
 
+export const Button = styled(QuietButton)`
+  ${ ({ theme }) => css`
+    padding: ${theme.space.half}px;
+    border-width: 0;
+    border-radius: 50%;
+    transition: none;
+    color: ${theme.colour.neutral[600]};
+    ${IconWrapper}{
+      width: 1.5em;
+      height: 1.5em;
+    }
+    ${IconWrapper} > svg{
+      height: 100%;
+      width: 100%;
+    }
+
+    &:hover{
+      color: ${theme.colour.neutral[600]};
+      transform: scale(1);
+    }
+    &::before {
```

Comment:
this is a lot of wiz-bang to only appear in the context of a table. Why is the table button so different form other buttons?

Reply:
It is the only circular (square) shaped button. Here I suggested a special hover treatment for this button and that treatment only works for icon only or circular (square) shaped buttons.

Reply:
And if we decide that this is the way to go we can consolidate this style with the rest of the buttons. This was more an exploration that anything else.